ecosystem.jenkins.config.js

```diff
@@ -31,14 +31,42 @@ module.exports = {
 			env: {
 				PORT: '9901',
 				SERVICE_BROKER: 'redis://localhost:6379/0',
-				SERVICE_GATEWAY_VOLATILE: 'redis://localhost:6379/3',
-				STRICT_READINESS_CHECK: false,
-				WS_RATE_LIMIT_ENABLE: false,
+				SERVICE_GATEWAY_REDIS_VOLATILE: 'redis://localhost:6379/3',
+				ENABLE_HTTP_API: 'http-status,http-version3,http-exports',
+				ENABLE_WS_API: 'blockchain,rpc-v3',
+				GATEWAY_DEPENDENCIES: 'indexer,connector',
```

Comment:
The variables are mentioned here to help our users to easily identify the available environment variables and their default values so that they need not have to manually go though the implementation to find every detail out.

Reply:
True, the code part is not directly visible to the user. Also, even if the user remove these dependencies from here, gateway won't start with out indexer,connector which could be a bit confusing. do you think removing the default from code would be better?

Reply:
Let's discuss internally and create another issue to make any further changes that we might need. Also, it is safe to assume that all the users will always be running the indexer and the connector microservices.